service/templates/projects.html

```diff
@@ -999,6 +999,85 @@ <h3>Virtual Workstations Commands</h3>
+            <div class="col-lg-4">
+              <input class="form-control" type="text" ng-model="settingsFormData.build_baron_settings.ticket_create_project" placeholder="Ticket create project">
+            </div>
+          </div>
+          <div class="form-group">
+            <div class="col-lg-4"> <label class="control-label"> Ticket search project </label> </div>
+          </div>
+          <div id="ticket_search_projects" class="form-group" ng-repeat="(index, project) in settingsFormData.build_baron_settings.ticket_search_projects">
+            <div class="col-lg-4"> <label class="control-label">[[project]]</label> </div>
+            <div class="col-lg-2">
+              <button class="btn btn-default btn-danger" type="button" ng-click="removeTicketSearchProject(index)">
+                <i class="fa fa-trash"></i>
+              </button>
+            </div>
+          </div>
+          <div class="form-group">
+            <div class="col-lg-4">
+              <input ng-model="ticket_search_project" class="form-control" type="text" placeholder="EVG">
+            </div>
+            <div class="col-lg-2">
+              <button class="plus-button btn btn-primary" ng-disabled="!(ticket_search_project)" type="button" ng-click="addTicketSearchProject()">
+                <i class="fa fa-plus"></i>
+              </button>
+            </div>
+          </div>
+
+          <div class="h3">Task Annotation Settings</div>
+          <div class="form-group">
+            <div class="col-lg-3"> <label class="control-label"> JIRA custom field </label> </div>
+            <div class="col-lg-3"> <label class="control-label"> JIRA custom display text </label> </div>
+          </div>
+          <div id="jira_custom_fields" class="form-group" ng-repeat="(index, jiraField) in settingsFormData.task_annotation_settings.jira_custom_fields">
+            <div class="col-lg-6"> <label class="control-label">[[jiraField]]</label> </div>
+            <div class="col-lg-2">
+              <button class="btn btn-default btn-danger" type="button" ng-click="removeJiraField(index)">
+                <i class="fa fa-trash"></i>
+              </button>
+            </div>
+          </div>
+          <div class="form-group">
+            <div class="col-lg-3">
+              <input class="form-control" ng-model="jira_field" type="text" placeholder="field">
+            </div>
+            <div class="col-lg-3">
+              <input class="form-control" ng-model="jira_display_text" type="text" placeholder="display text">
+            </div>
+            <div class="col-lg-2">
+              <button class="plus-button btn btn-primary" ng-disabled="!(jira_field) || !(jira_display_text)" type="button" ng-click="addJiraField()">
+                <i class="fa fa-plus"></i>
+              </button>
+            </div>
+          </div>
+          <div class="form-group">
+            <div class="col-lg-4"> <label class="control-label"> Webhook </label> </div>
```

Comment:
I think we should put a small description under this but I'm not actually sure what it would be; @malikchaya2 any suggestions?

Reply:
That's a good idea. 
Maybe something like: 

To use a custom setup for creating build failure tickets, specify a webhook endpoint and secret.